src/assisted_test_infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -316,12 +318,15 @@ def _delete_virsh_resources(cls, *filters):
         virsh_cleanup.clean_virsh_resources(skip_list=skip_list, resource_filter=filters)
 
     def prepare_nodes(self):
+        self._params = self._terraform_params(**self._config.get_all())
```

Comment:
if ``_params`` is something that should always dynamically infer its value from ``_config``, maybe we should make it a property:
```
@property
def _params(self):
    return self._terraform_params(**self._config.get_all())
```

Otherwise it's hard to remember having this line whenever we're using it (think of new methods added to the controller class)

Reply:
I thought doing that, but we don't want to get it dynamically every time we want to access some variable, that is why we save params value on protected property and updating it only when destroying/creating nodes.
I'll add method for getting params from config. that might be better